File: sem1_cs/AnmolSingh_225HSBB004/Assignment12_27_11_22/c/q2.c
/*Write a program to compare two strings and print the common letters in both. Without case sensitive.*/

#include<stdio.h>
#include<string.h>

int main(){

	char string1[50], string2[50], common[50];
	int len1,len2,i,j,k;

	printf("Enter first string\n");
	scanf("%s",string1);

	printf("Enter second string\n");   
        scanf("%s",string2);

	len1=strlen(string1);
	len2=strlen(string2);
	
	k=0;
	for(i=0;i<len1;i++)
	{	
		for(j=0;j<len2;j++)
	{	
		if((string1[i]==string2[j]) || (string1[i]==(string2[j]-32))||(string1[i]==(string2[j]+32)))
		{	
			common[k]=string1[i];
			k++;
		}	
	}	
	}
	printf("The common letters are:\n");
	for(i=0; i<strlen(common);i++){
		printf("%c ",common[i]);
	}
	
}

File: sem1_cs/AnmolSingh_225HSBB004/Assignment12_27_11_22/python/q2.py
#!/usr/bin/python3

#Write a python code to compare two lists of numbers. If a common element is found, then print that
#element and its position in both the lists.

def enter_list():
    lst=list()
    for i in range(20):
        num=input()
        if (num==''):
          return lst
        num=(int)(num)
        lst.append(num)
        
    return lst
        


print("Enter first list of numbers, max 20 elements: Enter empty string when done")
list1=enter_list()


print("Enter second list of numbers, max 20 elements:Enter empty string when done")
list2=enter_list()

print("List 1:\n",list1)
print("List 2:\n",list2)
index1=0
for num1 in list1:
    index2=0
    for num2 in list2:
        if(num1==num2):
            print("Common elements:")
            print(num1," in Position: List1: ", index1," List2: ", index2)
        index2=index2+1
    index1=index1+1



